sdk/identity/azure-identity/src/main/java/com/azure/identity/EnvironmentCredentialBuilder.java

```diff
@@ -39,8 +39,8 @@ public class EnvironmentCredentialBuilder extends CredentialBuilderBase<Environm
     private String authorityHost;
 
     /**
-     * Specifies the Azure Active Directory endpoint to acquire tokens.
-     * @param authorityHost the Azure Active Directory endpoint
+     * Specifies the Microsoft Entra endpoint to acquire tokens.
```

Comment:
Should this say, `Microsoft Entra ID endpoint` ?

Reply:
No. It's a feature/concept, so the ID suffix should be omitted. See https://learn.microsoft.com/azure/active-directory/fundamentals/new-name#glossary-of-updated-terminology.